libs/filamat/src/eiff/ShaderEntry.h

```diff
@@ -17,14 +17,16 @@
 #ifndef TNT_FILAMAT_SHADER_ENTRY_H
 #define TNT_FILAMAT_SHADER_ENTRY_H
 
+#include <string>
+
 namespace filamat {
 
 // TextEntry stores a shader in ASCII text format, like GLSL.
 struct TextEntry {
     uint8_t shaderModel;
     uint8_t variant;
     uint8_t stage;
-    char* shader;
+    std::string shader;
     size_t shaderSize;
```

Comment:
isn't `shaderSize` irrelevant if you're using an `std::string`?

Reply:
Yup good catch, meant to remove that.